app/coffee/RealTimeRedisManager.coffee

```diff
@@ -27,6 +28,8 @@ module.exports = RealTimeRedisManager =
 				catch e
 					return callback e
 				updates.push update
+				# record metric for updates removed from queue
+				metrics.summary "redis.pendingUpdates", jsonUpdate.length, {status: "pop"}
```

Comment:
Let's track the metric before the try block -- it might bail out in the catch section.

Reply:
Better yet, use a second loop before this one.

Reply:
It should be unlikely but I'll make a separate loop for simplicity (originally had it like that, then decided to combine them!).